packages/valory/skills/abstract_round_abci/tests/test_behaviours_utils.py

```diff
@@ -1259,6 +1260,10 @@ def test_wait_until_transaction_delivered_failed(self, *_: Any) -> None:
         )
         try_send(gen, success_response)
 
+    @pytest.mark.skipif(
+        platform.system() == "Windows",
+        reason="https://github.com/valory-xyz/open-autonomy/issues/1477",
+    )
```

Comment:
separate issue: https://github.com/valory-xyz/open-autonomy/issues/1477

note that this was also skipped on main before #1468